libcloud/compute/drivers/ec2.py

```diff
@@ -189,6 +189,41 @@
         'disk': 160000,
         'bandwidth': None
     },
+    'm4.large': {
+        'id': 'm4.large',
+        'name': 'Large Instance',
+        'ram': 8192,
+        'disk': 0,
```

Comment:
Do these templates have a disk specification? the other sizes above have the total disk size, these entries all have 0